server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -384,13 +385,13 @@ export default class ActivitiesMigrationController {
         startedDate,
       )}) .. datetime(${ActivitiesMigrationController.toISODateTime(endedDate)}))
       | where customDimensions.migrationId startswith '${migrationId}'
-      | where (name endswith 'failed' or name endswith 'ignored' or name endswith 'error')
+      | where (name endswith 'failed' or name endswith 'error')
```

Comment:
Don't include ignored migrations in failures. I got confused because I saw an ignore that recorded as a failure because I didn't clear the DLQ before the migration.